.github/workflows/ci.yml

```diff
@@ -73,9 +91,14 @@ jobs:
       run: .github/workflows/install_deps.sh amici pysb
 
     - name: Run tests
+      if: steps.filter.outputs.petab_filter == 'true'
```

Comment:
I like the conditional tests for pull requests. For commits on `develop` and `main`, I'd prefer running them unconditionally, though.

Reply:
that could be a solution for this. Not entirely sure how to set this as an additional condition but will check 👍🏼